lib/scanner/kv-scanner/tests/test_kv_scanner.c

```diff
@@ -139,14 +139,21 @@ typedef struct Testcase_t
   KVContainer expected;
 } Testcase;
 
-KVScanner *
+static KVScanner *
 create_kv_scanner(const ScannerConfig config)
```

Comment:
`config` variable is copied onto stack (-> `const` is not necessary here, or, pass a pointer to `ScannerConfig` instance).